server/test/unit/server/db/test_manage.py

```diff
@@ -166,48 +166,6 @@ def test_ensure_database_indexes_throws_exception(self, mock_model, mock_plugin_
                                                     "not allowed because the platform handlesit "
                                                     "for you.")
 
-    @patch.object(manage, 'ensure_database_indexes')
-    @patch('logging.config.fileConfig')
-    @patch('pkg_resources.iter_entry_points', iter_entry_points)
-    @patch('pulp.server.db.manage.connection.initialize')
-    @patch('pulp.server.db.manage.factory')
-    @patch('pulp.server.db.manage.logging.getLogger')
-    @patch('pulp.server.db.manage.RoleManager.ensure_super_user_role')
-    @patch('pulp.server.db.manage.managers.UserManager.ensure_admin')
-    @patch('pulp.server.db.migrate.models.pulp.server.db.migrations',
-           migration_packages.platform)
-    @patch('sys.argv', ["pulp-manage-db"])
-    @patch.object(models.MigrationPackage, 'apply_migration')
-    def test_admin_is_ensured(self, apply_migration, ensure_admin, ensure_super_user_role,
```

Comment:
I spent 10 minutes trying to make this "unit" test work, but it's really an integration test that mocks the whole world. Its test case is covered by smash, and it's real hard to work with, so I don't think it's worth keeping.